src-java/floodlight-service/floodlight-modules/src/main/java/org/openkilda/floodlight/kafka/RecordHandler.java

```diff
@@ -1424,6 +1424,7 @@ private void dumpMeters(SwitchId switchId, String correlationId, String replyToT
         try {
             logger.debug("Get all meters for switch {}", switchId);
             ISwitchManager switchManager = context.getSwitchManager();
+            // FIXME(surabujin): return null in case of errors
```

Comment:
don't get this fixme, null for the errors seems bad idea, also take into account this one https://github.com/telstra/open-kilda/pull/3160

Reply:
ok, it's just composed badly then. It should be get rid of returning null.